Microsoft.Toolkit.Uwp.Input.GazeInteraction/GazeStats.cpp

```diff
@@ -0,0 +1,47 @@
+//Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license.
+//See LICENSE in the project root for license information.
+
+#include "pch.h"
+#include "GazeStats.h"
+
+using namespace Platform::Collections;
+
+BEGIN_NAMESPACE_GAZE_INPUT
+
+GazeStats::GazeStats(int maxHistoryLen)
```

Comment:
Perhaps I'm missing something but it doesn't look like this class is used anywhere?